nx-dev/nx-dev/pages/recipes/[...segments].tsx

```diff
@@ -96,6 +100,9 @@ export const getStaticProps: GetStaticProps = async ({
     return {
       props: {
         document,
+        widgetData: {
+          githubStarsCount: await fetchGithubStarCount(),
```

Comment:
At some point, let's improve this to be a serverless function run daily that would update the statically generated codebase.

Reply:
yeah I'm not sure it matters. Since we show rounded to 100 stars it won't matter I think even if it shows less on the website than it is for real on GH. We have enough updates to the docs and rebuilds of the website I think.